### PR TITLE
make formula lists more readable and formatted

### PR DESCRIPTION
**Pull request by Moisan**
Follow-up of https://github.com/Homebrew/homebrew-core/pull/173129

The characters escapes `\` where mixed up with newlines `\`.

### CODE REVIEW

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -219,12 +219,92 @@ jobs:
               content: system "swift", "build"
 
             - label: long build
-              path: Formula/.+/(agda|apache-pulsar|arangodb|aws-sdk-cpp|cp2k|deno|dotnet|emscripten|envoy|gcc|ghc|graph-tool|gstreamer|haskell-language-server|libtensorflow|mame|metashell|mlkit|mpich|node|nwchem|ponyc|pytorch|rust|swift|texlive|qt|root|souffle|v8|vtk|)(@[0-9]+)?.rb
+              path: >
```
Comment (carlocab) [inline .github/workflows/triage.yml:222]: Using `>` will replace the newlines with spaces, so this doesn't work, unfortunately 😞

Comment (Moisan) [reply .github/workflows/triage.yml:222]: :( but `path: |` would work?

Comment (carlocab) [reply .github/workflows/triage.yml:222]: Nope, `|` retains new lines: https://yaml-multiline.info

Comment (Bo98) [reply .github/workflows/triage.yml]: We could probably just strip/collapse new lines in the action. Real file names shouldn't really have them.

Comment (Moisan) [reply .github/workflows/triage.yml]: Reading trough the yaml documentation, I think we want `>-` which strips newlines and doesn't add one at the end.

Comment (carlocab) [reply .github/workflows/triage.yml]: `>-` just strips the final `\n`, but intermediate ones are still replaced with spaces.

Comment (Moisan) [reply .github/workflows/triage.yml]: Ok so I think the only option left is to keep the double quote string and double escape characters within the string.

**Review by MikeMcQuaid — APPROVED** [review]

**Review by carlocab — APPROVED** [review]
Worth a try.

**Comment by carlocab** [comment]
Thanks @Moisan!

**Pull request merged**